engine/src/main/java/com/datatorrent/stram/StramClient.java

```diff
@@ -448,6 +450,14 @@ public void startApplication() throws YarnException, IOException
       }
       String libJarsCsv = copyFromLocal(fs, appPath, localJarFiles.toArray(new String[]{}));
 
+      if (UserGroupInformation.isSecurityEnabled() && customKeystoreConfig != null && customKeystorePath != null) {
+        String[] customSSLConfArr = {customKeystoreConfig, customKeystorePath};
+        String customSSLFileConf = copyFromLocal(fs, appPath, customSSLConfArr);
+        String configFileName = (new java.io.File(customKeystoreConfig)).getName();
+        dag.getAttributes().put(Context.DAGContext.STRAM_HTTP_CUSTOM_CONFIG, configFileName);
```

Comment:
@sanjaypujare I think "superfluous" is very much appropriate for this. What you cite as example to the contrary actually supports this, it uses "custom" when describing how to create a keystore, not as the name of a property to reference it. Be that is it may, would you look for examples of the same spelling error elsewhere or rather fix it?